### PR TITLE
Configuration tuning

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/HSEARCH-3481
 * https://hibernate.atlassian.net/browse/HSEARCH-3482

### CODE REVIEW

File: legacy/documentation/src/main/asciidoc/elasticsearch-integration.asciidoc [hunk 1]
```diff
@@ -179,11 +179,11 @@ Configuration is minimal.
 Add the configuration properties to your `persistence.xml` or where you put the rest of your Hibernate Search configuration.
 
 Select Elasticsearch as the backend:: `hibernate.search.default.indexmanager elasticsearch`
-Hostname and port for Elasticsearch:: `hibernate.search.default.elasticsearch.host \http://127.0.0.1:9200` (default)
```
Comment (yrodiere) [inline legacy/documentation/src/main/asciidoc/elasticsearch-integration.asciidoc]: Please don't change the legacy documentation... ?

Comment (gsmet) [reply legacy/documentation/src/main/asciidoc/elasticsearch-integration.asciidoc]: Argh, silly me, will fix.

Comment (gsmet) [reply legacy/documentation/src/main/asciidoc/elasticsearch-integration.asciidoc]: Fixed.

**Comment by yrodiere** [comment]
Merged, thanks

**Pull request merged**